lib/imagery/iscatt_core.c

```diff
@@ -687,19 +690,21 @@ int I_compute_scatts(struct Cell_head *region, struct scCats *scatt_conds,
     const char *mapset;
     char header[1024];
 
-    int fd_cats_rasts[scatt_conds->n_a_cats];
-    FILE *f_cats_rasts_conds[scatt_conds->n_a_cats];
+    int *fd_cats_rasts =
+        G_malloc(scatt_conds->n_a_cats * sizeof(scatt_conds->n_a_cats));
```

Comment:
size of int?

Reply:
Really the same, but sizeof(int) is a bit cleaner, shorter... changed.